README.md

```diff
@@ -109,7 +109,8 @@ Currently we don't have a playbook which installs KubeVirt on a Kubernetes clust
 ### OpenShift cluster
 
 Install KubeVirt on your OpenShift cluster.
-[KubeVirt parameters doc](https://github.com/rthallisey/kubevirt-ansible/blob/master/roles/kubevirt/README.md)
+
+[KubeVirt parameters doc](https://github.com/kubevirt/kubevirt-ansible/blob/master/roles/kubevirt/README.md)
```

Comment:
Don't write `doc` and similar in documentation please. It's a slang world.
The explanation of options/parameters/variables goes after a command.
I would replace `KubeVirt parameters doc` with something like `[See](link here) the list of used parameters.`